packages/api/src/routers/svc/speech.ts

```diff
@@ -0,0 +1,113 @@
+import express from 'express'
+import cors from 'cors'
+import { corsConfig } from '../../utils/corsConfig'
+import { getRepository } from '../../entity/utils'
+import { getPageById } from '../../elastic/pages'
+import { synthesizeTextToSpeech } from '../../utils/textToSpeech'
+import { Speech, SpeechState } from '../../entity/speech'
+import { UserPersonalization } from '../../entity/user_personalization'
+import { buildLogger } from '../../utils/logger'
+import { getClaimsByToken } from '../../utils/auth'
+
+const logger = buildLogger('app.dispatch')
+
+export function speechServiceRouter() {
+  const router = express.Router()
+
+  router.options('/', cors<express.Request>({ ...corsConfig, maxAge: 600 }))
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  router.post('/', async (req, res) => {
+    logger.info('Speech svc request', {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      body: req.body,
+    })
+    const token = req.query.token as string
+    try {
+      if (!(await getClaimsByToken(token))) {
+        logger.info('Unauthorized request', { token })
+        return res.status(200).send('UNAUTHORIZED')
+      }
+    } catch (error) {
+      logger.error('Unauthorized request', { token, error })
+      return res.status(200).send('UNAUTHORIZED')
+    }
```

Comment:
Added authentication here

Reply:
Does the user token get passed when this is called?